packages/react/src/UnorderedList/UnorderedList.tsx

```diff
@@ -10,18 +10,27 @@ import { UnorderedListItem } from './UnorderedListItem'
 
 export type UnorderedListProps = {
   markers?: boolean
+  inverseColor?: boolean
```

Comment:
This is only a reorder.

```suggestion
  inverseColor?: boolean
  markers?: boolean
```